.github/workflows/check-image-version.yaml

```diff
@@ -19,13 +19,13 @@ jobs:
         run: |
           git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
           git config --global user.name "${GITHUB_ACTOR}"
-          
+
           git fetch origin
           git checkout -B Update-image-version-${{ inputs.branch }} origin/${{ inputs.branch }}
 
       - name: Check and update images
         run: |
-          IMAGES=("registry.access.redhat.com/ubi9/go-toolset" "registry.access.redhat.com/ubi9/ubi-minimal")
+          IMAGES=("registry.access.redhat.com/ubi9/go-toolset" "registry.access.redhat.com/ubi9/ubi-minimal", "registry.redhat.io/rhel9/mariadb-105", "registry.access.redhat.com/ubi9/httpd-24", "registry.redhat.io/ubi9/python-311")
```

Comment:
```suggestion
          IMAGES=("registry.access.redhat.com/ubi9/go-toolset" "registry.access.redhat.com/ubi9/ubi-minimal" "registry.redhat.io/rhel9/mariadb-105" "registry.access.redhat.com/ubi9/httpd-24" "registry.redhat.io/ubi9/python-311")
```

The `,` are causing an issue when pulling the image so they need to be removed, there also needs to be a step to authenticate with registry.redhat.io as its causing issues.

Error:
```
Run IMAGES=("registry.access.redhat.com/ubi9/go-toolset" "registry.access.redhat.com/ubi9/ubi-minimal", "registry.redhat.io/rhel9/mariadb-105", "registry.access.redhat.com/ubi9/httpd-24", "registry.redhat.io/ubi9/python-311")
time="2023-12-[15](https://github.com/JasonPowr/timestamp-authority/actions/runs/7220053346/job/19672231673#step:4:16)T09:06:07Z" level=fatal msg="Error parsing image name \"docker://registry.access.redhat.com/ubi9/ubi-minimal,:latest\": invalid reference format"
time="2023-12-15T09:06:07Z" level=fatal msg="Error parsing image name \"docker://registry.redhat.io/rhel9/mariadb-105,:latest\": invalid reference format"
time="2023-12-15T09:06:07Z" level=fatal msg="Error parsing image name \"docker://registry.access.redhat.com/ubi9/httpd-24,:latest\": invalid reference format"
time="2023-12-15T09:06:07Z" level=fatal msg="Error parsing image name \"docker://registry.redhat.io/ubi9/python-311:latest\": unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication"
```